notesapi/v1/models.py

```diff
@@ -27,13 +28,14 @@ def create(cls, note_dict):
         if len(note_dict) == 0:
             raise ValidationError('Note must have a body.')
 
-        ranges = note_dict.get('ranges', list())
+        ranges = note_dict.get('ranges', [])
```

Comment:
Because @dan-f asked me to. Really I have absolutely no preference. Happy to change it back. I think then I would also use list() for tags.